.github/workflows/preflight-release.yml

```diff
@@ -77,13 +81,18 @@ jobs:
           sed -i "s|quay.io/opdev/preflight:[0-9]*\.[0-9]*\.[0-9]*|quay.io/opdev/preflight:${PREFLIGHT_LATEST}|" roles/preflight/defaults/main.yml
           sed -i "s|quay.io/opdev/preflight:[0-9]*\.[0-9]*\.[0-9]*|quay.io/opdev/preflight:${PREFLIGHT_LATEST}|" roles/preflight/README.md
           git add roles/preflight/defaults/main.yml roles/preflight/README.md
-          git commit --allow-empty -m "Update Preflight to release $PREFLIGHT_LATEST"
+          git commit -s -m "Update Preflight to release $PREFLIGHT_LATEST"
           git push --set-upstream origin preflight-release-$PREFLIGHT_LATEST
 
       - name: Create PR moving to the latest Preflight
         if: steps.compare_versions.outputs.new_release == 'true' && steps.check_branch.outputs.branch_exists == 'false'
         env:
           PREFLIGHT_LATEST: ${{ steps.check_release.outputs.preflight_latest }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        # using echo in to properly format new line chars
         run: |
-          gh pr create --title "Move to Preflight release $PREFLIGHT_LATEST" --body "This pull request was opened by a bot.\n\nCheckDallasWorkload: preflight-green" --base main --head preflight-release-$PREFLIGHT_LATEST
+          gh pr create \
+            --title "Move to Preflight release $PREFLIGHT_LATEST" \
+            --body "$(echo -e "This pull request was opened by a bot.\n\nCheckDallasWorkload: preflight-green")" \
```

Comment:
@ramperher here is what has worked to ensure the new lines in the [PR description](https://github.com/redhatci/ansible-collection-redhatci-ocp/pull/346).

Reply:
I saw that, thanks for taking care!